consensus/expected.go

```diff
@@ -57,6 +58,10 @@ type TicketSigner interface {
 	SignBytes(data []byte, signerAddr address.Address) (types.Signature, error)
 }
 
+// DefaultBlockTime is the estimated proving period time.
+// We define this so that we can fake mining in the current incomplete system.
```

Comment:
This comment is a little confusing to me.  `DefaultBlockTime` is defined for more than just faking the VDF, right?  Also, is the terminology "proving period" correct?  I always associate that with a storage miner.

Reply:
I think when the comment was written we assumed that this constant would go away, since then the spec has change s.t. we need it. Will update.

Reply:
I'm pretty sure it's nothing to do with PoSt proving period.